src/components/projects/project-edit-form/project-edit-container/project-edit-details/project-edit-details.tsx

```diff
@@ -0,0 +1,182 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Project, User } from 'utils';
+import { Col, Container, Row, Form, InputGroup } from 'react-bootstrap';
+import PageBlock from '../../../../shared/page-block/page-block';
+import { wbsPipe, endDatePipe, fullNamePipe } from '../../../../../shared/pipes';
+import { WbsElementStatus } from 'utils/lib/types/project-types';
+
+// new parts added at the bottom
+interface projectDetailsProps {
+  project: Project;
+  users: User[];
+}
+
+const ProjectEditDetails: React.FC<projectDetailsProps> = ({ project, users }) => {
+  const statuses = Object.values(WbsElementStatus).filter((status) => status !== project.status);
+  const startDate =
+    project.workPackages.length > 0
+      ? project.workPackages
+          .reduce(
+            (min, cur) => (cur.startDate < min ? cur.startDate : min),
+            project.workPackages[0].startDate
+          )
+          .toLocaleDateString()
+      : 'n/a';
+  const endDate =
+    project.workPackages.length > 0
+      ? endDatePipe(
+          project.workPackages.reduce(
+            (min, cur) => (cur.startDate < min ? cur.startDate : min),
+            project.workPackages[0].startDate
+          ),
+          project.workPackages.reduce((tot, cur) => tot + cur.duration, 0)
+        )
+      : 'n/a';
+
+  const editDetailsInputBuilder = (
+    title: string,
+    type: string,
+    defaultValue: any,
+    prefix = '',
+    suffix = '',
+    placeholder = '',
+    readOnly = false
+  ) => {
+    const formInput = (
+      <Form.Group>
+        <InputGroup>
+          {prefix ? <InputGroup.Text>{prefix}</InputGroup.Text> : <></>}
+          <Form.Control
+            type={type}
+            defaultValue={defaultValue}
+            placeholder={placeholder}
+            readOnly={readOnly}
+          />
+          {suffix ? <InputGroup.Text>{suffix}</InputGroup.Text> : <></>}
+        </InputGroup>
+      </Form.Group>
+    );
+
+    return (
+      <>
+        <b>{title}</b>
+        {formInput}
+        <br />
+      </>
+    );
+  };
+
+  const statusSelect = (
+    <Form.Control as="select">
+      <option key={0} value={project.status}>
+        {project.status}
+      </option>
+      {statuses.map((status, index) => (
+        <option key={index + 1} value={status}>
+          {status}
+        </option>
+      ))}
+    </Form.Control>
+  );
+
+  const buildUsersSelect = (title: string, defaultUser: User) => {
+    const otherUsers = users.filter((user) => {
+      return fullNamePipe(user) !== fullNamePipe(defaultUser);
+    });
+    return (
+      <>
+        <b>{title}</b>
+        <Form.Control as="select">
+          <option key={0} value={fullNamePipe(defaultUser)}>
```

Comment:
let's make the key and maybe the value be the user's ID because that should be unique and is what will need to be sent to the back-end. it's only the text inside the option tag that is displayed to the user I believe and that can be the human-readable name